src/Pages/Login/Login.js

```diff
@@ -0,0 +1,248 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router-dom';
+import styled from 'styled-components';
+import { KAKAO_LOGIN_API } from '../../config';
+// import './Login.scss';
+
+class Login extends Component {
+  kakaoLoginHandler = () => {
+    window.Kakao.Auth.login({
+      success: (auth) => {
+        console.log(auth);
+        fetch(KAKAO_LOGIN_API, {
+          method: 'POST',
+          headers: {
+            Authorization: auth.access_token,
+          },
+        })
+          .then((res) => res.json())
+          .then((res) => {
+            if (res.token) {
+              localStorage.setItem('Log_Token', res.token);
+              console.log('로그인 되었습니다.');
+            } else {
+              console.log('다시 확인해 주세요.');
+            }
+          });
+      },
+      fail: function (err) {
+        console.log('a');
+        console.log('에러', err);
+      },
+    });
+  };
+
+  render() {
+    return (
+      <div>
+        <Article>
+          <LoginBox>
+            <LoginForm>
+              <LoginInner>
+                <HEAD>로그인</HEAD>
```

Comment:
```suggestion
                <Head>로그인</Head>
```
- 전체 대문자 네이밍은 보통 상수 변수에 많이 사용하고, 컴포넌트는 일반적으로 위와 같이 사용합니다.

Reply:
상수에만 확인이요!